tests/src/Simple/Generics/Generics.cs

```diff
@@ -1627,11 +1643,18 @@ private static void TestInstanceFields()
             }
         }
 
+        private static void TestUnconstructedTypes()
+        {
+            var a = typeof(UnconstructedTypeInstantiator<UnconstructedTypeWithGCStatics>).ToString();
```

Comment:
Maybe add a comment that this is testing for compilation failures due to references to unused static bases with a link to the github issue?